docker-compose.yml

```diff
@@ -38,6 +38,12 @@ services:
       - ./tmp/nexus-data:/nexus-data:z
     ports:
       - 8082:8081
+    # Nexus needs more open files, else it will throw errors.  The Cachito integrtion tests expose the problem,
+    # a java.io.exception of too many open files will be thrown.
+    ulimits:
+      nofile:
+        soft: 65536
+        hard: 65536
```

Comment:
Fixed typo.

Reply:
This PR is for adding metrics endpoint for Prometheus monitoring. However, these lines fix issue for the Nexus container, it would be good to move this fix to a separate commit. You can create a new PR for this separate commit, or it is ok to be included in this one if you think they are relative.

Reply:
Ok, new PR created, removed from here.